src/scripts/main.js

```diff
@@ -1 +1,74 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  let resolved = false;
```

Comment:
This variable `resolved` is used to check if the promise is resolved or not. But it's not necessary to use this variable. Because if the promise is resolved, it will not be able to be resolved or rejected again. The Promise object will ignore any other attempts to resolve or reject after the first call.